packages/expo-ui/ios/ExpoUIModule.swift

```diff
@@ -19,5 +19,15 @@ public class ExpoUIModule: Module {
     View(TextInputView.self)
     View(ProgressView.self)
     View(GaugeView.self)
+
+    View(SwiftUIContainer.self)
```

Comment:
Just so I understand:
Do you consider this a sample and don't want to land these additional components, or do you want to ship them in `@expo/ui`?

If yes, I'd try to reuse existing components (button, picker, ect) where possible – sort of how unwrappedchildren do it. We can always define some subcomponent that we provide in swiftUI that is then used when rendering a child from a SwiftUI parent.

Reply:
this is just a poc. if the pure swiftui components are in the right track. i would like to keep it inside expo-ui package. maybe we can have a separate import like `@expo/ui/swift-ui`.
i can follow up to share these view implementations.